src/App.css

```diff
@@ -18,7 +18,64 @@
   font-size: large;
 }
 
+#text-input {
+  margin-top: 100px;
+}
+
 @keyframes App-logo-spin {
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+ul {
+  list-style: none;
+}
+
+#shopping-cart-logo {
```

Comment:
I'm used to using classes vs ids to do css styling. We should decide on one way of doing this for the project so everything is consistent.